install/ansible/tasks/bootstrap_ubuntu.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on Ubuntu"
+  package:
+    name: "{{ item }}"
+    state: latest
+  with_items:
```

Comment:
What about `nsenter`? And also for centos.

Reply:
nsenter is part of util-linux already ?

Reply:
Missed that. Sorry.
If so, it means that this won't support trusty, let's keep in mind that we should document it or add a comment here.